### ISSUE TITLE
Get app icon

### ISSUE THREAD

**Issue by naranik** [klattimer/LGWebOSRemote#156]
Hi,

Does anyone know how to get the icon associated with an app? If you send the listApps request you get back a list of apps each of which has a URL for an icon, something like 'http://{host}:3000/resources/{hex digits}/{filename}.png'. A normal HTTP request gives a 404 error and I've tried a few different requests using the websocket.

Thanks

**Comment by klattimer** [comment]
Seems like bad implementation on LGs part. If a url has a proper protocol and address in an api response it should work.

Maybe we need someone to confirm it doesn't work on other models? 

Please note your software version in the bug, might not be helpful otherwise.

**Comment by naranik** [comment]
Yes it's very weird to return an HTTP URL and then not allow it to be fetched by an HTTP request especially when the logos are really not secret.
